longest_common_substring.py

```diff
@@ -0,0 +1,15 @@
+def longest_common_substring(X: str, Y: str) -> int:
+    m, n = len(X), len(Y)
```

Comment:
follow coding standard and guideline

Reply:
I have made the changes. Kindly review it